File: proxy.js
const { ProxyAgent } = require('undici');

const url = 'https://ipv4.icanhazip.com';
const client = new ProxyAgent(
	'http://:dgQk19rhe9IfT5dQ:HbzWVsg8EIRrQsyj@_country-id@geo.iproyal.com:12321'
);
const proxyTest = async () => {
	try {
		const response = await fetch(url, {
			dispatcher: client,
		});

		const data = await response.text();
		console.log(data);
	} catch (error) {
		console.error(error);
	}
};

proxyTest();

File: proxy.py
import requests

url = 'https://ipv4.icanhazip.com'
proxy = 'http://_country-id@geo.iproyal.com:12321'
proxy_auth = 'dgQk19rhe9IfT5dQ:HbzWVsg8EIRrQsyj'
proxies = {
    'http': f'http://{proxy_auth}@{proxy}',
    'https': f'http://{proxy_auth}@{proxy}'
}

response = requests.get(url, proxies=proxies)
print(response.text)